src/api/views.py

```diff
@@ -99,6 +100,22 @@ def available(request, domain=""):
     return json_response
 
 
+@require_http_methods(["GET"])
+@login_not_required
+# Since we cache domain RDAP data, cache time may need to be re-evaluated this if we encounter any memory issues
+@ttl_cache(ttl=600)
+def rdap(request, domain=""):
+    """Returns JSON dictionary of a domain's RDAP data from Cloudflare API"""
+    domain = request.GET.get("domain", "")
```

Comment:
overall lgtm! I was just wondering why the ttl is 600 seconds? Is that standard?

Reply:
I think generally 5 minutes is one of the standard defaults for caching which we also use in the availability endpoint! If there ends up being a cache time that makes more sense for this data we can also always update it if the need arises :")

Reply:
Okay, LGTM. 600 seconds is 10 minutes

Reply:
Ack you're right, did not math that math correctly :")